docs/GCS-batchsink.md

```diff
@@ -26,6 +26,12 @@ Properties
 ----------
 **Reference Name:** Name used to uniquely identify this sink for lineage, annotating metadata, etc.
 
+**Use Connection** Whether to use a connection, if a connection is used,
+the credentials does not need to be provided.
```

Comment:
@itsankit-google Can we rewrite as "you do not need to provide the credentials."